### ISSUE TITLE
[template] Subtitle tracking issue template

### ISSUE THREAD

**Issue by LucileDT** [jlm2017/jlm-video-subtitles#1]
# Important note

Usually, the tracking issues will be created automatically by @RobotInsoumis. You should not have to create them yourself, unless you're captioning an old video.

# Template
## Issue title
`[subtitles] [language] VIDEO TITLE`

`language` should be a 2-char [ISO 639-1](https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-1) identifier, such as `fr`, `en`, `de`, `zh`, etc.

## Issue description
Please copy and paste the following text to create a new transcription tracking issue:

``` markdown
## VIDEO TITLE

&nbsp;          | Info
--------------- | ---------------
**Date**        | Mardi 21 février 2017
**Durée**       | 0:01:43 :clock7:
**Langue**      | Français :fr:
**Vidéo**       | [Voir dans YouTube :arrow_upper_right:](https://www.youtube.com/watch?v=aNw579fuoXM)
**Sous-titres** | [Éditer dans YouTube :arrow_upper_right:](https://www.youtube.com/timedtext_editor?v=aNw579fuoXM&tab=captions&bl=vmp&action_mde_edit_form=1&lang=fr&ui=hd)
```

*Anything prefixed by # will be displayed in bulk, bold and underlined, as a level 1 title.*

**Comment by Ricyeam** [comment]
can you change to "Anglais" instead of "Anglais (Etas-Unis) " ? I know it's just a model, but to ensure consistency between videos it would better.

**Comment by LucileDT** [comment]
Mise à jour du modèle : ce sera plus facile pour vous de créer de nouvelles issues à partir de celle-ci :D

**Comment by LucileDT** [comment]
J'ai détaillé un exemple + changé le titre de l'issue pour la rendre plus explicite.

**Comment by LucileDT** [comment]
I changed language from French to English, to use this template in all new subtitles languages (German, Spanish, Portugese, Chinese).

**Comment by Goutte** [comment]
Pour la langue dans le titre de l'issue, on devrait se mettre d'accord sur une sémantique, ça nous permettra plus facilement de trier les issues !

Je propose [ISO 639-1](https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-1), ie. deux lettres significatives _(fr, en, de, zn, ...)_

Exemple : `[subtitles] [en] La vidéo d'un Insoumis`

**Comment by LucileDT** [comment]
Oui, c'était l'idée, mais je ne connais pas les codes pour tous les pays malheureusement :(
